scripts/create-aks-cluster.sh

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 : "${CLUSTER_NAME:?Environment variable empty or not defined.}"
 
 get_random_region() {
-    local REGIONS=("eastus" "eastus2" "westus2" "westeurope" "uksouth" "northeurope" "francecentral")
+    local REGIONS=("eastus" "eastus2" "southcentralus" "westeurope" "uksouth" "northeurope" "francecentral")
```

Comment:
`westus2` has been flaky lately due to lack of cores so switching it to southcentralus instead.